app/assets/javascripts/mobile/mobile_comments.js

```diff
@@ -121,72 +162,73 @@
-      var self = this;
-      $.ajax({
-        url: link.attr("href"),
-        beforeSend: function(){
-          link.addClass("loading");
-        },
-        context: link,
-        success: function(data) {
-          self.appendCommentBox.call(this, link, data);
-        },
-        error: function() {
-          link.removeClass("loading");
-        }
-      });
-    },
-
-    appendCommentBox: function(link, data) {
-      link.removeClass("loading");
-      link.removeClass("inactive");
-      var bottomBar = link.closest(".bottom_bar").first();
-      bottomBar.append(data);
+      var bottomBar = link.closest(".bottom-bar").first();
       var textArea = bottomBar.find("textarea.comment_box").first()[0];
+      bottomBar.find(".add-comment-switcher").removeClass("hidden");
       autosize(textArea);
     },
 
     updateStream: function(form, data) {
-      var bottomBar = form.closest(".bottom_bar").first();
+      var bottomBar = form.closest(".bottom-bar").first();
       this.addNewComments(bottomBar, data);
       this.updateCommentCount(bottomBar);
-      this.updateReactionCount(bottomBar);
+      this.increaseReactionCount(bottomBar);
       this.handleCommentShowing(form, bottomBar);
       bottomBar.find("time.timeago").timeago();
     },
 
     addNewComments: function(bottomBar, data) {
-      if ($(".comment_container", bottomBar).length === 0) {
-        $(".show_comments", bottomBar).after($("<div/>", {"class": "comment_container"}));
-        $(".comment_container", bottomBar).append($("<ul/>", {"class": "comments"}));
+      if ($(".comment-container", bottomBar).length === 0) {
+        $(".show-comments", bottomBar).after($("<div/>", {"class": "comment-container"}));
+        $(".comment-container", bottomBar).append($("<ul/>", {"class": "comments"}));
       }
-      $(".comment_container .comments", bottomBar).append(data);
+      $(".comment-container .comments", bottomBar).append(data);
     },
 
     // Fix for no comments
     updateCommentCount: function(bottomBar) {
-      var commentCount = bottomBar.find(".comment_count");
+      var commentCount = bottomBar.find(".comment-count");
       commentCount.text(commentCount.text().replace(/(\d+)/, function (match) {
-        return parseInt(match) + 1;
+        return parseInt(match, 10) + 1;
       }));
     },
 
     // Fix for no reactions
-    updateReactionCount: function(bottomBar) {
-      var toggleReactionsLink = bottomBar.find(".show_comments").first();
-      toggleReactionsLink.text(toggleReactionsLink.text().replace(/(\d+)/, function (match) {
-        return parseInt(match) + 1;
-      }));
+    increaseReactionCount: function(bottomBar) {
+      var toggleReactionsLink = bottomBar.find(".show-comments").first();
+      var count = toggleReactionsLink.text().match(/.*(\d+).*/);
+      var text = "";
+
+      // No previous comment
+      if(!count){
+        text = Diaspora.I18n.t("stream.reactions.one", {count: "1"});
```

Comment:
Please have another look at the example I gave. To use plurals you don't specify which one you would like to have (`zero`, `one` or `other`) and instead just give the key and the variable, so in this case
```js
text = Diaspora.I18n.t("stream.reactions", {count: 1});
```
You also don't have to convert the integer to a string for the variable.